src/main/java/todolist/Application/CreateTask/CreateTask.java

```diff
@@ -15,8 +16,8 @@ public CreateTask(TaskRepository taskRepository) {
 
   public CreateTaskResponse execute(CreateTaskRequest createTaskRequest) {
 
-    Task createdTask = new Task(
-        UUID.randomUUID(),
+    Task createdTask = Task.create(
+        new TaskId(),
```

Comment:
I think Task should be responsible of creating the TaskId by itself, but we can talk about that.